src/client/src/orangehrmRecruitmentPlugin/pages/ViewCandidatesList.vue

```diff
@@ -193,24 +196,27 @@ export default {
 
   setup() {
     const {$t} = usei18n();
+    const {locale} = useLocale();
+    const {jsDateFormat} = useDateFormat();
+    const {$tEmpName} = useEmployeeNameTranslate();
+
     const candidateDataNormalizer = data => {
       return data.map(item => {
         return {
           id: item.id,
           vacancy: item.vacancy?.name,
-          candidate: `${item.firstName} ${item.middleName || ''} ${
-            item.lastName
-          }`,
-          manager: item.vacancy
-            ? `${item.vacancy.hiringManager.firstName} ${
-                item.vacancy.hiringManager.lastName
-              } ${
-                item.vacancy.hiringManager.terminationId
-                  ? $t('general.past_employee')
-                  : ''
-              }`
+          candidate: `${item.firstName} ${item.middleName} ${item.lastName}`,
```

Comment:
if candidate's middle name is not available, it will return null. That will show null as the middle name then.

Reply:
Done.